README.md

```diff
@@ -55,6 +55,12 @@ can be written as:
 [:div>p>b "Nested Element"]
 ```        
 
+The `:class` attribute accepts collections in addition to plain string.
```

Comment:
Suggestion: improve language.

"The :class attribute can accept either a collection or a string"